packages/access-api/src/models/delegations.js

```diff
@@ -163,6 +163,18 @@ export function createDelegationRowUpdate(d) {
   }
 }
 
+/**
+ * @param {Ucanto.Delegation} d
+ * @returns {DelegationRowUpdate<import('../types/access-api-cf-db').DelegationsV3Row>}
+ */
+export function createDelegationRowUpdateV3(d) {
+  return {
+    cid: d.cid.toV1().toString(),
```

Comment:
> and delegations can be encoded in DAG-PB.

is this a typo?

Reply:
> Looks like CIDs will retain original base encoding when parsed so if you want to normalize you should pass base32 as an argument.

good catch. I thought it would normalize to base32 for all v1. I will do it explicitly.

Reply:
I added tests asserting the cid kinds for cid column in d1 as well as the cid part of r2 keys https://github.com/web3-storage/w3protocol/pull/578/commits/e379699486b621aa1e87b407f4d30fd95e510799#diff-80f191e42400b9b26178ff477cf9edac5ac2e8106246b9be714368ed719525baR134